.github/workflows/python-threatexchange-release.yaml

```diff
@@ -29,13 +29,13 @@ jobs:
         run: |
           python ./setup.py sdist bdist_wheel
       - name: Publish distribution to Test PyPI
-        uses: pypa/gh-action-pypi-publish@master
+        uses: pypa/gh-action-pypi-publish@main
```

Comment:
We're sure the gh-action are also on main now?

Reply:
nope, wanted to see what errored, looks like `@release/v1` is prefered so I updated. https://github.com/pypa/gh-action-pypi-publish